docs/index.md

```diff
@@ -4,54 +4,31 @@ page_description: Introduces the Docker Registry
 page_keywords: registry, images, repository
 IGNORES-->
 
-# Docker Registry 2.0
+# Docker Registry
 
-Docker Registry stores and distributes images centrally. It's where you push images to and pull them from; Docker Registry gives team members the ability to share images and deploy them to testing, staging and production environments.
+The Registry is a stateless, highly scalable server side component that stores and lets you distribute Docker images.
 
-Docker provides a hosted registry as part of [Docker Hub](https://hub.docker.com). Docker Hub is a cloud service that securely manages your images. It features organization accounts, automated builds, and much, much more.
+Users looking for a ready-to-go solution are encouraged to head-over to the [Docker Hub](https://hub.docker.com), which provides a free-to-use, hosted Registry, plus additional features (organization accounts, automated builds, and more).
 
-Docker Registry is the core technology behind the Docker Hub. You can run your own registry instance if you want to host your images privately. Docker Registry features:
+On the other hand, people interested in finer grained integration, or more control over the content they publish, should run and operate their own Registry.
 
- - **Pluggable storage drivers**: Images can be stored in Amazon S3, Microsoft Azure or the local filesystem.
- - **Webhook notifications**: When an image is pushed to your registry, webhooks can fire off to launch CI builds, send notifications to IRC, etc.
+## Understanding the Registry
 
-To get started with your own Docker Registry, head over to the instructions on how to [deploy a registry](deploying.md).
+A Registry is a storage and content delivery system, holding named Docker images, available in different tagged versions. For example, the image `distribution/registry`, with tags `2.0` and `latest`.
```

Comment:
If "A Registry" here refers to a registry "in general", not the "Docker" registry, I don't think it should be capitalised.